activesupport/test/time_zone_test.rb

```diff
@@ -725,7 +725,7 @@ def test_all_uninfluenced_by_time_zone_lookups_delegated_to_tzinfo
     assert_not_includes all_zones, galapagos
   end
 
-  def test_all_not_raises_exception_with_mizzing_tzinfo_data
+  def test_all_doesnt_raise_exception_with_missing_tzinfo_data
```

Comment:
LOL, how did I miss that - thanks 👍

Reply:
thanks for the review. will we backport this to `5-2-stable`?

Reply:
Yes - I'll cherry pick

Reply:
Done in e4ab3c17203cd90ec07ca78dc25b5989705bce16 - thanks again